### DISCUSSION TITLE
Gyroscope based parallax

### DISCUSSION THREAD

**Discussion by 59xa** [ppy/osu#13022] [Ideas] [answered]
**Proposal:**
I have this idea where the background image does an effect as you change your phone’s angle. It would be a good addition for the current parallax effect we have right now.
**Example:** https://youtu.be/WBp77LHAdho

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by CenTdemeern1** [comment]
I would like to second this, because the parallax is very jumpy on touch devices due to the lack of a smooth moving cursor.

**Comment by bdach** [comment, marked as answer]
#6864
